reference/orders.v3.yml

```diff
@@ -416,6 +416,50 @@ paths:
         required: true
         schema:
           type: string
+  '/stores/{store_hash}/v3/orders/payment_actions/refunds/{refund_id}':
+    get:
+      summary: Return a refund by refund ID.
+      description: |-
+        Returns a refund by refund ID.
+        Requires at least one of the following scopes:
+        * `store_v2_orders`
+        * `store_v2_transactions`
+      operationId: RefundID_Get
+      parameters:
+        - name: refund_id
+          in: path
+          description: Refund ID.
+          required: true
+          schema:
+            type: integer
+        - name: store_hash
+          in: path
+          required: true
+          schema:
+            type: string
+        - $ref: '#/components/parameters/Accept'
+        - $ref: '#/components/parameters/Content-Type'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RefundID_Get'
+        required: true
+        x-examples: null
+      responses:
+        '201':
+          description: Return a refund with 'Refund_ID'.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/RefundID_Get'
```

Comment:
`data` is an object, not an array.

Example response:

<img width="1047" alt="image" src="https://user-images.githubusercontent.com/63274600/172734137-a4f6830c-e745-42a6-ac59-e889dec5b620.png">